core/arch/arm/plat-nuvoton/main.c

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2016-2023, Linaro Limited
+ * Copyright (c) 2014-2023, STMicroelectronics International N.V.
+ * Copyright (C) 2022-2023 Nuvoton Ltd.
+ */
+
+#include <console.h>
+#include <drivers/gic.h>
+#include <drivers/ns16550.h>
+#include <kernel/boot.h>
+#include <kernel/linker.h>
+#include <platform_config.h>
+#include <trace.h>
+
+#if defined(CFG_NPCM_DEBUG)
```

Comment:
comment not addressed
(but maybe you really prefer this extra useless pre-compilation directives)

Reply:
agree, much cleaner